WordPressKit/WordPressComRestApi.swift

```diff
@@ -443,6 +453,9 @@ extension WordPressComRestApi {
         ]
 
         let mappedError = errorsMap[errorCode] ?? WordPressComRestApiError.unknown
+        if mappedError == .invalidToken {
+            invalidTokenHandler?()
```

Comment:
I'm seeing that this path is properly exercised via `WordPressComRestApiTests.testInvalidTokenFailedCall()`.